web/packages/teleport/src/lib/tdp/client.ts

```diff
@@ -347,6 +348,9 @@ export default class Client extends EventEmitterWebAuthnSender {
         },
         (responseFrame: ArrayBuffer) => {
           this.sendRDPResponsePDU(responseFrame);
+        },
+        (data: ImageData, hotspot_x: number, hotspot_y: number) => {
```

Comment:
Technically, given how we're using it in the Rust wasm, this should be
```suggestion
        (data?: ImageData, hotspot_x?: number, hotspot_y?: number) => {
```